front/components/sparkle/navigation.tsx

```diff
@@ -190,23 +190,24 @@ export const subNavigationBuild = ({
     variant: "secondary",
     menus: dataSourceItems,
   });
-
-  nav.push({
-    id: "developers",
-    label: isAdmin(owner) ? "Developers" : null,
-    variant: "secondary",
-    menus: [
-      {
-        id: "developers",
-        label: "Developer Tools",
-        icon: CommandLineIcon,
-        href: `/w/${owner.sId}/a`,
-        current: current === "developers",
-        subMenuLabel: current === "developers" ? subMenuLabel : undefined,
-        subMenu: current === "developers" ? subMenu : undefined,
-      },
-    ],
-  });
+  if (isAdmin(owner)) {
```

Comment:
This is inaccurate, this section is builder more than admin right?

Reply:
That can be discussed — The correction here is that as it was setup, the test changed only the SectionHeader but not the section.

IMO today, we want to display to both Admin and Builders (So I'll just remove the isAdmin)